qpc/messages.py

```diff
@@ -50,6 +50,14 @@
 
 CRED_LIST_NO_CREDS = 'No credentials exist yet.'
 
+CRED_BECOME_METHOD_HELP = 'The method to become for network privilege'\
+    ' escalation. Valid values: sudo, su, pbrun, pfexec, doas, dzdo,'\
+    ' ksu, runas.'
+CRED_BECOME_USER_HELP = 'The user to become when running a privileged'\
+    ' command during network scan.'
+CRED_BECOME_PASSWORD_HELP = 'The privilege escalation password to be'\
+    ' used when running a network scan.'
+
```

Comment:
@jlprevatt Would you look at the messages that I added and let me know if I need to update their format to match the changes that you made in qpc/messages.py ? Thank you!

Reply:
Global: In my edits on messages.py, there was originally inconsistency with the line continuation characters. I adopted the extra space at the end of the line (e.g., between words). What you have here is space at the beginning. Not a big deal. If there is a known Python best practice that spaces go at the beginning of a continued line, please let me know and I will aim for that in the future.
Line 57: "...during a network scan."